### DISCUSSION TITLE
Change in g.dart when we use @UseRowClass in moor

### DISCUSSION THREAD

**Discussion by salman731** [simolus3/drift#1261] [Q&A]
I want to ask that what will be change in g.dart generated file when I use @UseRowClass on table class.

**Comment by simolus3** [comment]
Did you see the documentation [here](https://moor.simonbinder.eu/docs/advanced-features/custom_row_classes/)? When you use `@UseRowClass`, moor will stop generating a row class by itself. Instead, it uses a class written by you to store data from that table. Other code like companions is still generated as before.
